h3/Cargo.toml

```diff
@@ -19,11 +19,15 @@ categories = [
     "web-programming::http-server",
 ]
 
+[features]
+allow_access_to_core = []
```

Comment:
That's fantastic.

Makes it a lot more clear that this is a no-no feature rather than something that sounds like a good "normal" feature to enable

Reply:
I love this!!

Reply:
Great. This would still break if we do a breakage in the now exposed backend, if `h3-webtransport` is present in the dependency tree due to cargo feature resolution.

However, `h3-webtransport` *should* usually reside near the top in your local crates and not external crates.io dependencies. So fixing these issues would be far easier.

The feature has now been renamed